### PR TITLE
Added API's for Alert

### PR DESCRIPTION
**Pull request by chandrashekhar-macrometa**
New API's were added in alert.

The Api's were added for Create, Update, Get

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
     <groupId>co.macrometa</groupId>
     <artifactId>c84j</artifactId>
```
[the remark this answers is not in the document]
Comment (grainier-macrometa) [reply pom.xml:5]: @StoyanTc - I think @chandrashekhar-macrometa has already released [c84j-1.1.14](https://github.com/Macrometacorp/c84j/releases/tag/c84j-1.1.14). And, this change is coming from the mvn-release-plugin... However, I think it's better to get the PR approved, before the release.

File: src/main/java/com/c8db/C8DB.java [hunk 2]
```diff
@@ -1015,4 +1016,29 @@ public synchronized C8DB build() {
...
+    /**
+     * Gets Alerts
+     * @param queryParamMap
+     * @return
+     * @throws C8DBException
+     */
+    Collection<AlertEntity>  getAlerts(Map<String, String> queryParamMap) throws C8DBException;
+
+    /**
+     * Acknowledges or Resolves an alert
```
Comment (yehan-macrometa) [inline src/main/java/com/c8db/C8DB.java:1028]: @chandrashekhar-macrometa - The description implies that only `acknowledged` and `resolved` can be updated.  Is that correct?

File: src/main/java/com/c8db/entity/AlertEntity.java [hunk 3]
```diff
@@ -0,0 +1,36 @@
+package com.c8db.entity;
```
Comment (grainier-macrometa) [inline src/main/java/com/c8db/entity/AlertEntity.java:1]: @chandrashekhar-macrometa - Let's add the license header.

File: src/main/java/com/c8db/internal/InternalC8DB.java [hunk 4]
```diff
@@ -256,6 +278,30 @@ protected Request getUserRequest(final String tenant, final String database, fin
         };
     }
 
+    protected ResponseDeserializer<Collection<AlertEntity>> getAlertsResponseDeserializer() {
```
Comment (grainier-macrometa) [inline src/main/java/com/c8db/internal/InternalC8DB.java:281]: @chandrashekhar-macrometa - `alertsResponseDeserializer` and `getAlertsResponseDeserializer` are bit confusing. Shall we rename them to give the idea of one returns a single entity deserializer while the other returns an entity collection deserializer?

**Review by grainier-macrometa — COMMENTED** [review]
@chandrashekhar-macrometa - Added a few comments...

**Comment by chandrashekhar-macrometa** [comment]
The Changes for this PR are merged as part of PR https://github.com/Macrometacorp/c84j/pull/45
Hence closing this PR

**Pull request closed without merging**